src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -47,6 +47,23 @@ public AuthenticationTests()
             }
         }
 
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // Managed implementation connects successfully.
```

Comment:
> I'm not sure I follow that: when I run on Windows with no parameters it seems to run against the HTTP.SYS component, not the managed one.

The src .csproj is set up to build the Windows implementation on Windows.  But with a one line change to the .csproj, you can have it build the managed implementation on Windows.  And if you do that, the tests will then be running against the managed implementation but still on Windows.  If we used [ActiveIssue(..., TestPlatforms.AnyUnix)] to say "there's a bug in the managed implementation, the managed implementation is what's used on unix, so disable this test for unix", then when we use the managed implementation on Windows, the test will run and fail.  Instead, we use a conditional fact/theory to say "only run this test if the windows implementation is being used".  We don't have a way currently to use ActiveIssue to do the same thing.

> Also, my question was more about the comment "Managed implementation connects successfully." That sounds to me like a bug in the managed implementation.

It is a bug in the managed implementation.  That's why the conditional fact/theory is disabling the test for the managed implementation by saying it should only run with the windows implementation.  The comment is describing what the bug is, but it should have included the issue number as well to help associate it.